GoogleSignIn/Sources/GIDToken.m

```diff
@@ -55,4 +55,37 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
   [encoder encodeObject:_expirationDate forKey:kExpirationDateKey];
 }
 
+#pragma mark - isEqual
+
+- (BOOL)isEqual:(id)object {
+  if (self == object) {
+    return YES;
+  }
+  if (![object isKindOfClass:[GIDToken class]]) {
+    return NO;
+  }
+  return [self isEqualToGIDToken:(GIDToken *)object];
+}
+
+- (BOOL)isEqualToGIDToken:(GIDToken *)other {
+  return [_tokenString isEqual:other.tokenString] &&
+      [self isTheSameDateWithDate1:_expirationDate date2:other.expirationDate];
+}
+
+- (BOOL)isTheSameDateWithDate1:(NSDate *)date1
```

Comment:
The parameter names shouldn't be numbered. Try to think of a more descriptive name.

Reply:
Done.